packages/Radio/src/components/Group/Group.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+
+const propTypes = {
+  /** aria-labelledby prop on the containing group element */
+  a11yLabelledByText: PropTypes.string,
```

Comment:
`a11yText` please (this is a way to insulate app developers from the implementation details).